### DISCUSSION TITLE
After updating SO, most service won't start and some are missing.

### DISCUSSION THREAD

**Discussion by pmzhou** [Security-Onion-Solutions/securityonion#1849] [Unsupported Versions] [answered]
Checking Docker status
                                                                                                                    
    Docker ----------------------------------------------------------------------------------------------- [ OK ]    

Checking container statuses
                                                                                                                    
    so-aptcacherng --------------------------------------------------------------------------------------- [ OK ]    
    so-cortex --------------------------------------------------------------------------------------- [ MISSING ]    
    so-curator ------------------------------------------------------------------------------------------- [ OK ]    
    so-dockerregistry ------------------------------------------------------------------------------------ [ OK ]    
    so-elastalert ------------------------------------------------------------------------------------- [ ERROR ]    
    so-elasticsearch ------------------------------------------------------------------------------------- [ OK ]    
    so-filebeat --------------------------------------------------------------------------------------- [ ERROR ]    
    so-fleet --------------------------------------------------------------------------------------------- [ OK ]    
    so-grafana ------------------------------------------------------------------------------------------- [ OK ]    
    so-idstools ------------------------------------------------------------------------------------------ [ OK ]    
    so-influxdb ------------------------------------------------------------------------------------------ [ OK ]    
    so-kibana ----------------------------------------------------------------------------------------- [ ERROR ]    
    so-kratos -------------------------------------------------------------------------------------------- [ OK ]    
    so-logstash ------------------------------------------------------------------------------------------ [ OK ]    
    so-mysql --------------------------------------------------------------------------------------------- [ OK ]    
    so-nginx ---------------------------------------------------------------------------------------- [ MISSING ]    
    so-playbook --------------------------------------------------------------------------------------- [ ERROR ]    
    so-redis --------------------------------------------------------------------------------------------- [ OK ]    
    so-sensoroni ----------------------------------------------------------------------------------------- [ OK ]    
    so-soc ----------------------------------------------------------------------------------------------- [ OK ]    
    so-soctopus ------------------------------------------------------------------------------------------ [ OK ]    
    so-steno ---------------------------------------------------------------------------------------- [ MISSING ]    
    so-strelka-coordinator ------------------------------------------------------------------------------- [ OK ]    
    so-strelka-filestream ----------------------------------------------------------------------------- [ ERROR ]    
    so-strelka-frontend ---------------------------------------------------------------------------------- [ OK ]    
    so-strelka-gatekeeper -------------------------------------------------------------------------------- [ OK ]    
    so-strelka-manager ----------------------------------------------------------------------------------- [ OK ]    
    so-suricata ------------------------------------------------------------------------------------------ [ OK ]    
    so-telegraf ------------------------------------------------------------------------------------------ [ OK ]    
    so-thehive ------------------------------------------------------------------------------------------- [ OK ]    
    so-thehive-es ------------------------------------------------------------------------------------- [ ERROR ]    
    so-wazuh --------------------------------------------------------------------------------------------- [ OK ]    
    so-zeek ---------------------------------------------------------------------------------------------- [ OK ]

**Comment by TOoSmOotH** [comment, marked as answer]
Try sudo so-docker-refresh

**Comment by pmzhou** [comment]
Wow. That restored my smile which was up-side-down.
Worked like a charm. Thanks, this saved me from re-installing.
________________________________
